在原有文档基础上重新翻译 (#136)
iomesh/docs: docs/deploy/install-iomesh.md

## Quick Installation Guide

For quick installation, run the corresponding script below according to your OS distribution.

> > **_NOTE_: If the script has been executed with Helm3 not installed yet, Helm3 will be re-installed automatically.**

Author: 这里我理解的是：一般情况下，运行以上脚本，包括 Helm3 在内的一些程序会被安装。如果运行了该脚本但是发现 Helm3 没有被安装，系统会再次自动安装 Helm3。Helm3 will be re-installed automatically if has not been found in the system. 看看改成这句可以吗？

> > **_NOTE_: If the script has been executed with Helm3 not installed yet, Helm3 will be re-installed automatically.**

Author: 这里我理解的是：一般情况下，运行以上脚本，包括 Helm3 在内的一些程序会被安装。如果运行了该脚本但是发现 Helm3 没有被安装，系统会再次自动安装 Helm3。Helm3 will be re-installed automatically if has not been found in the system. 看看改成这句可以吗

> > **_NOTE_: If the script has been executed with Helm3 not installed yet, Helm3 will be re-installed automatically.**

Author: 这里我理解的是：一般情况下，运行以上脚本，包括 Helm3 在内的一些程序会被安装。如果运行了该脚本但是发现 Helm3 没有被安装，系统会再次自动安装 Helm3。

Helm3 will be re-installed automatically if it is not installed in the system. 看看改成这句可以吗? 或者保留原句也可以。

> > **_NOTE_: If the script has been executed with Helm3 not installed yet, Helm3 will be re-installed automatically.**

Reviewer: 嗷嗷 原句可以的

> > **_NOTE_: If the script has been executed with Helm3 not installed yet, Helm3 will be re-installed automatically.**

Reviewer 2: @yuqiu123 原意是：如果 helm3 没有安装，脚本会自动安装 helm3。

这里没有 re-install 的意思

> > **_NOTE_: If the script has been executed with Helm3 not installed yet, Helm3 will be re-installed automatically.**

Author: > @yuqiu123 原意是：如果 helm3 没有安装，脚本会自动安装 helm3。
> 
> 这里没有 re-install 的意思

那还是保留原句的说法？

> > **_NOTE_: If the script has been executed with Helm3 not installed yet, Helm3 will be re-installed automatically.**

Reviewer 3: 此处保留原句描述

> > **_NOTE_: If the script has been executed with Helm3 not installed yet, Helm3 will be re-installed automatically.**

Author: **_NOTE_: Helm3 would be installed automatically if it is not founded.**

> > **_NOTE_: If the script has been executed with Helm3 not installed yet, Helm3 will be re-installed automatically.**

Reviewer 3: 这个地方需要解决一下。 re-install 换成 install

```

<!--RHEL8/CentOS8/CoreOS-->

```shell
# Every node running IOMesh must have an IP address belonging to IOMESH_DATA_CIDR
export IOMESH_DATA_CIDR=10.234.1.0/24; curl -sSL https://iomesh.run/install_iomesh_el8.sh | sh -
```
<!--END_DOCUSAURUS_CODE_TABS-->

Wait for minutes. IOMesh will be installed successfully if all IOMesh Cluster pods are running. 

```shell
watch kubectl get --namespace iomesh-system pods
```

IOMesh has been installed successfully!

> **_NOTE_: IOMesh resources installed by running the entered script will be reserved for future troubleshooting if any error occurs during installation. You may enter the script below to remove all IOMesh resources from the Kubernetes Cluster: `curl -sSL https://iomesh.run/uninstall_iomesh.sh | sh -`**

## Customized Installation Guide

For installation with customized configurations, follow the steps below.

### Install CSI Snapshotter

The [CSI snapshotter](https://github.com/kubernetes-csi/external-snapshotter) is part of Kubernetes implementation of Container Storage Interface (CSI). Install CSI Snapshotter to enable the Volume Snapshot feature.

> > **_NOTE_: CSI Snapshotter should be installed in each cluster.**

Author: 好的，那改为 CSI Snapshotter should be installed per cluster？

> > **_NOTE_: CSI Snapshotter should be installed in each cluster.**

Reviewer: 嗯嗯

> > **_NOTE_: CSI Snapshotter should be installed in each cluster.**

Reviewer 2: 这里想强调的是，每个 Cluster 只安装**一个**CSI Snapshotter

> > **_NOTE_: CSI Snapshotter should be installed in each cluster.**

Author: > 这里想强调的是，每个 Cluster 只安装**一个** CSI Snapshotter

Install only one CSI Snapshotter per cluster. 看看这句可以吗？

> > **_NOTE_: CSI Snapshotter should be installed in each cluster.**

Reviewer 3: > > 这里想强调的是，每个 Cluster 只安装**一个** CSI Snapshotter
> 
> Install only one CSI Snapshotter per cluster. 看看这句可以吗？

我觉得可以，per cluster 强调一下是 per K8s cluster

> > **_NOTE_: CSI Snapshotter should be installed in each cluster.**

Author: ```suggestion
> **_NOTE_: Install only one CSI Snapshotter per Kubernetes cluster.**
```

            mountIscsiLock: true
    ```

>    **(optional)** If you simply want IOMesh to utilize part of Kubernetes nodes, configure the lables of nodes to be utilized in `chunk.podPolicy.affinity`. For example:

Author: configure the labels

    NAMESPACE: iomesh-system
    STATUS: deployed
    REVISION: 1

>     TEST SUITE: none

Reviewer: 这个应该是固定的，不能修改。 @gxglls 确认下

Reviewer 3: 这里是程序执行后的输出，不需要修改

    iomesh-openebs-ndm-svp9n                              1/1     Running   0          2m8s
    iomesh-zookeeper-0                                    1/1     Running   0          2m3s
    iomesh-zookeeper-1                                    1/1     Running   0          102s
    iomesh-zookeeper-2                                    1/1     Running   0          76s
    iomesh-zookeeper-operator-7b5f4b98dc-6mztk            1/1     Running   0          2m8s
    operator-85877979-66888                               1/1     Running   0          2m8s
    operator-85877979-s94vz                               1/1     Running   0          2m8s
    operator-85877979-xqtml                               1/1     Running   0          2m8s
    ```

IOMesh Cluster is installed successfully. Please go to [Setup IOMesh](setup-iomesh.md).

[1]: http://iomesh.com/charts
[2]: http://www.iomesh.com/docs/installation/setup-iomesh-storage#setup-data-network

## Install IOMesh on OpenShift Container Platform

You may also install and use IOMesh through the IOMesh Operator on the OperatorHub page of the Red Hat OpenShift Container Platform.

### Pre-Install

> Run the script below to install the dependencies of IOMesh Operator, then set up IOMesh specifications and settings for the OpenShift Cluster. Note that the script should be executed in an environment where oc or kubectl can access the OpenShift Cluster. 

Author: ```suggestion
Run the script below, which will install the dependencies of the IOMesh Operator and configure IOMesh specifications and settings for the OpenShift Cluster.
```

> Run the script below to install the dependencies of IOMesh Operator, then set up IOMesh specifications and settings for the OpenShift Cluster. Note that the script should be executed in an environment where oc or kubectl can access the OpenShift Cluster. 

Author: ```suggestion
Run the script below, which will install the dependencies of the IOMesh Operator and configure IOMesh specifications and settings for the OpenShift Cluster. Note that the script should be executed in an environment where oc or kubectl can access the OpenShift Cluster. 
```

    operator-85877979-xqtml                               1/1     Running   0          2m8s
    ```

IOMesh Cluster is installed successfully. Please go to [Setup IOMesh](setup-iomesh.md).

[1]: http://iomesh.com/charts
[2]: http://www.iomesh.com/docs/installation/setup-iomesh-storage#setup-data-network

## Install IOMesh on OpenShift Container Platform

You may also install and use IOMesh through the IOMesh Operator on the OperatorHub page of the Red Hat OpenShift Container Platform.

### Pre-Install

> Run the script below to install the dependencies of IOMesh Operator, then set up IOMesh specifications and settings for the OpenShift Cluster. Note that the script should be executed in an environment where oc or kubectl can access the OpenShift Cluster. 

Reviewer: 前半句有点奇怪。这个脚本会包括「安装依赖项」和「进行必要的配置」。

Author: 1. Run the script below to install the dependencies of IOMesh Operator and configure IOMesh specifications and settings for the OpenShift Cluster.

Author: Run the script below, which will install the dependencies of the IOMesh Operator and configure IOMesh specifications and settings for the OpenShift Cluster. 看看这句可以吗？

Reviewer: 嗯 我觉得这样更合适
